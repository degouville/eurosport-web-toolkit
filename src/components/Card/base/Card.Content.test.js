import React from 'react';
import { shallow } from 'enzyme';
import CardContent from './Card.Content';

it('renders a CardContent', () => {
  expect(shallow(<CardContent>CardContent</CardContent>)).toMatchSnapshot();
});
