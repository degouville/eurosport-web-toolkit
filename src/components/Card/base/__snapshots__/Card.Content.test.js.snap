// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a CardContent 1`] = `
<div>
  Content
</div>
`;
